models/classes/Platform/Repository/LtiPlatformRepositoryInterface.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA
+ */
+
+namespace oat\taoLti\models\classes\Platform\Repository;
+
+use Countable;
+use oat\taoLti\models\classes\Platform\LtiPlatform;
+
+/**
+ * Service methods to manage the LTI platform business objects.
+ */
+interface LtiPlatformRepositoryInterface extends Countable
```

Comment:
I'm not quite sure what's the purpose of this interface?
We should probably literally implement https://github.com/oat-sa/lib-lti1p3-core/blob/master/src/Registration/RegistrationRepositoryInterface.php, without introducing an interface of our own, unless we need an adapter implementation for some reason.

Reply:
It's an interface for Platforms, not Registrations. This interface may be extended in the future with non-RDF implementation with no impact on the Registration repo.